entrypoint/microservice/routes/documents.py

```diff
@@ -0,0 +1,219 @@
+# All routes related to /documents endpoint
+# We will provide the following routes:
+#   /documents
+#   /documents/id
+#   /documents/id/similar
+#   /documents/id/similarity_update
+#   /documents/retrieve
+
+import sys
+import requests
+from flask import (
+    Blueprint, flash, g, redirect, request, session, url_for, jsonify, current_app as app
+)
+from ..config import config_db
+from werkzeug.exceptions import abort
+
+bp = Blueprint('documents', __name__, url_prefix='/api/v1/documents')
+
+def get_documents_from_db(document_ids):
+    """
+    Function receives a list of document ids and returns a list of dictionaries of documents data.
+
+    Parameters:
+        documents_ids : list(int)
+            list of document ids
+    
+    Returns:
+        success (boolean), list of dictionaries of document data if the extraction from the database was successful.
+    """
+
+    db = config_db.get_db()
+    if db.cursor is None:
+        return False, {'Error' : 'The connection could not be established'}
+    
+    statement = "SELECT * FROM documents WHERE document_id IN %s;"
+    try:
+        db.cursor.execute(statement, (tuple(document_ids),))
+    except:
+        return False, {'Error' : 'You provided invalid document ids.'}
+    
+    # Enumerating the fields
+    num_fields = len(db.cursor.description)
+    field_names = [i[0] for i in db.cursor.description]
+    documents = [{ field_names[i]: row[i] for i in range(num_fields) } for row in db.cursor.fetchall()]
+
+    # Cleaning the output:
+    # - removing fulltext field
+    # - slicing down the fulltext_cleaned field to 500 chars
+    # - we return only the first 10 results
+    for i in range(len(documents)):
+        if documents[i]['fulltext_cleaned'] is not None:
+            documents[i]['fulltext_cleaned'] = documents[i]['fulltext_cleaned'][:500]
+        documents[i].pop('fulltext')
+    config_db.close_db()
+
+    return True, documents[:10]
+
+
+@bp.route('/', methods=['GET'])
+def get_documents():
```

Comment:
Should support filtering options:

```
{
    "document_ids": [list of documents ids given],
    "types": "types of documents, if reasonable",
    "languages": "languages of the documents",
    "locations": "locations of the documents",
    "informea": "informea tags associated with the documents",
}
```

Reply:
I agree, but this seems like it would be better to implement in another PR.